components/gitpod-db/src/typeorm/workspace-db-impl.ts

```diff
@@ -327,9 +327,12 @@ export abstract class AbstractTypeORMWorkspaceDBImpl implements WorkspaceDB {
         ).map(wsinfo => wsinfo.latestInstance);
     }
 
-    public async findRunningInstancesWithWorkspaces(installation?: string, userId?: string): Promise<RunningWorkspaceInfo[]> {
+    public async findRunningInstancesWithWorkspaces(installation?: string, userId?: string, excludeStopping: boolean = true): Promise<RunningWorkspaceInfo[]> {
```

Comment:
I'm a tiny bit worries about optional booleans that have a default value `true`. E.g., what if this value ever becomes `undefined`, `null`, or other "falsy" value?

A safer option could be to have default value `false`:

```suggestion
    public async findRunningInstancesWithWorkspaces(installation?: string, userId?: string, includeStopping: boolean = false): Promise<RunningWorkspaceInfo[]> {
```

Reply:
Yeah, I get the point. I don't usually like adding an exclude flag. The TS compiler should protect against it being set as `undefined` _et al_, but I'm not against changing it